handlers/user.go

```diff
@@ -757,7 +816,7 @@ func UserNotificationsCenter(w http.ResponseWriter, r *http.Request) {
 
 		val, ok := validatorMap[sub.EventFilter]
 		if !ok {
-			if (utils.GetNetwork() == "mainnet" && strings.HasPrefix(string(sub.EventName), "monitoring_")) || strings.HasPrefix(string(sub.EventName), utils.GetNetwork()+":"+"monitoring_") {
```

Comment:
Monitoring events don't have a network prefix.

Reply:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/b4c340c7c8061b05363ba04e586dab99e37d7b3b/handlers/user.go#L575-L579

This looks like a bug then. The function (`UserUpdateMonitoringSubscriptions`) is not used though so I suggest to remove it together with https://github.com/gobitfly/eth2-beaconchain-explorer/blob/945fa921a93d006bc6113a7a4d3d709e993c8e49/cmd/explorer/main.go#L629.